core/Updater/Migration/Db/DropColumn.php

```diff
@@ -17,10 +17,6 @@ public function __construct($table, $columnName)
     {
         $sql = sprintf("ALTER TABLE `%s` DROP COLUMN `%s`", $table, $columnName);
 
-        if (!empty($placeColumnAfter)) {
-            $sql .= sprintf(' AFTER `%s`', $placeColumnAfter);
-        }
-
         parent::__construct($sql, static::ERROR_CODE_COLUMN_NOT_EXISTS);
```

Comment:
This change doesn't seem to fit with the rest. Was it intentional?

Reply:
as he had written in the PR description:

> I modify core/Updater/Migration/Db/DropColumn.php with delete unnecessary code cause placeColumnAfter never exist in this scope

Reply:
Sorry, didn't see that